doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -50,51 +50,47 @@ Synopsis
 ========
 In general, the command to mount CephFS via kernel driver looks like this::
 
-    mount -t ceph {device-string}:{path-to-mounted} {mount-point} -o {key-value-args} {other-args}
+    mount -t ceph {device-string}={path-to-mounted} {mount-point} -o {key-value-args} {other-args}
 
 Mounting CephFS
 ===============
 On Ceph clusters, CephX is enabled by default. Use ``mount`` command to
 mount CephFS with the kernel driver::
 
     mkdir /mnt/mycephfs
-    mount -t ceph :/ /mnt/mycephfs -o name=foo
+    mount -t ceph <name>@<fs_name>[.<fsid>]=/ /mnt/mycephfs -o mon_host=<mon_host>
 
-The key-value argument right after option ``-o`` is CephX credential;
-``name`` is the username of the CephX user we are using to mount CephFS. The
-default value for ``name`` is ``guest``.
+``name`` is the username of the CephX user we are using to mount CephFS. ``fs_name``
+is the file system to mount. Monitor hosts are passed as an option parameter.
 
 The kernel driver also requires MON's socket and the secret key for the CephX
 user. In case of the above command, ``mount.ceph`` helper figures out these
 details automatically by finding and reading Ceph conf file and keyring. In
 case you don't have these files on the host where you're running mount
 command, you can pass these details yourself too::
 
-    mount -t ceph 192.168.0.1:6789,192.168.0.2:6789:/ /mnt/mycephfs -o name=foo,secret=AQATSKdNGBnwLhAAnNDKnH65FmVKpXZJVasUeQ==
+    mount -t ceph foo@cephfs=/ /mnt/mycephfs -o mon_host=192.168.0.1:6789,secret=AQATSKdNGBnwLhAAnNDKnH65FmVKpXZJVasUeQ==
 
-Passing a single MON socket in above command works too. A potential problem
-with the command above is that the secret key is left in your shell's command
-history. To prevent that you can copy the secret key inside a file and pass
-the file by using the option ``secretfile`` instead of ``secret``::
+A potential problem with the command above is that the secret key is left in your
+shell's command history. To prevent that you can copy the secret key inside a file
+and pass the file by using the option ``secretfile`` instead of ``secret``::
 
-    mount -t ceph :/ /mnt/mycephfs -o name=foo,secretfile=/etc/ceph/foo.secret
+    mount -t ceph foo@cephfs=/ /mnt/mycephfs -o mon_host=192.168.0.1:6789,secretfile=/etc/ceph/foo.secret
 
 Ensure the permissions on the secret key file are appropriate (preferably,
 ``600``).
 
-In case CephX is disabled, you can omit ``-o`` and the list of key-value
-arguments that follow it::
+Multiple monitor hosts can be passed as follows::
 
-    mount -t ceph :/ /mnt/mycephfs
+  mount -t ceph foo@cephfs=/ /mnt/mycephfs -o mon_host="192.168.0.1:6789 192.168.0.2:6789",secretfile=/etc/ceph/foo.secret
```

Comment:
Having to quote because of spaces here is a bit cumbersome. Might be good to pick a new delimiter to use between the monaddrs.